README.md

```diff
@@ -125,6 +125,13 @@ For the mock configuration files, it is mostly the local repository that you hav
 
 For the json file, the paths of mock have to match what is in the mock configuration files, and so does the repo. The paths for patches and routines can be relative, so it should work straight out of the checkout
 
+These are all the places where the `DIRACOS_REPO` is referred:
+
+  * https://github.com/DIRACGrid/DIRACOS/blob/master/mockConfigs/mock-build-diracos.cfg#L92
+  * https://github.com/DIRACGrid/DIRACOS/blob/master/mockConfigs/mock-install-diracos.cfg#L9
+  * https://github.com/DIRACGrid/DIRACOS/blob/master/mockConfigs/mock-install-diracos.cfg#L94
+  * https://github.com/DIRACGrid/DIRACOS/blob/master/config/diracos.json#L9
+
```

Comment:
I would add: "If you do not change the above mentioned lines, you have to have `$DIRACOS_REPO=/diracos_repo`, otherwise the compilation will fail."